mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/134 - component view actions

Closes #134 

To test:
- Run any scenario
- Open ComponentView and select computer
- Click Scan and get message dialog about computer's state
- Click Diagnose and check CyberCIEGE process window for message about unknown message (still waiting on server changes to actually implement this functionality)
- Click Reimage, accept confirmation, and make sure funds go down
- Click Replace Drive, accept confirmation, and make sure funds go down
